### ISSUE TITLE
Capitialization

### ISSUE THREAD

**Issue by MarisaKirisame** [deech/fltkhs#82]
fl_show_colormapC is inconsistent with other (should be Fl_show_colormapC
fl_utf8C also have this problem

also fl_utf8C is empty file

**Comment by deech** [comment]
Yeah. :( The C code is a mess but I don't think that's exposed on the Haskell side. Do you need functions from those files?

**Comment by MarisaKirisame** [comment]
No. I just make and found problem.
